data/intuos-m-p3-wl.tablet

```diff
@@ -21,7 +21,7 @@
 Name=Wacom Intuos BT M
 ModelName=CTL-6100WL
 DeviceMatch=usb:056a:0378;bluetooth:056a:0379
-Class=Bamboo
+Class=Intuos3
```

Comment:
I think it's better to use the `EvdevCodes` here. The `Class` tag is too ambiguous, not used outside of libwacom and mapping this to a 'random' other class just to hope for the right behaviour is bound to break at some point. `EvdevCodes` is the proper solution for tablets that send events on different codes than expected.

Reply:
Yeah, this definitely makes sense. I removed the Class and put in the evdev codes in [d66563b](https://github.com/linuxwacom/libwacom/pull/261/commits/d66563b5e6e0b4de0d9cefc7c234d278fafa56cb).

Reply:
sorry, I was imprecise here: we still need the `Class=` line, it'll just end up being unused.